coins/monero/Cargo.toml

```diff
@@ -55,6 +55,7 @@ monero-generators = { path = "generators", version = "0.1" }
 
 [dev-dependencies]
 tokio = { version = "1", features = ["full"] }
+monero-rpc = "0.3.2"
```

Comment:
This locks monero-rpc to only 0.3.2. Using "0.3" would allow automatically updating to the most recent bug fix.